composer.json

```diff
@@ -18,15 +18,15 @@
     "require": {
         "neos/neos": "4.3.x-dev",
         "neos/nodetypes": "4.3.x-dev",
-        "neos/demo": "@dev",
         "neos/site-kickstarter": "4.3.x-dev",
 
-        "neos/neos-ui": "@dev",
+        "neos/demo": "~5.0.0",
+        "neos/neos-ui": "^3.3",
+        "neos/seo": "~2.1",
+        "neos/redirecthandler-neosadapter": "~3.0.0",
```

Comment:
Why only bugfix releases for the redirect plugins  but minors for neos /seo?
Also just bugfixes for neos/demo

Reply:
@gerhard-boden streamligned the definition. Now only Neos.Demo uses a defined minor version. That makes sense so at least the visible neos you get is the same for the 4.3 branch.